Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -665,7 +665,6 @@ enum class CryptoAlgorithmIdentifierTag {
     AES_KW = 11,
     HMAC = 12,
     SHA_1 = 14,
-    SHA_224 = 15,
```

Comment:
It's probably worth adding a comment indicating 15 used to be used for SHA_224.  There may be binary blobs on people's disks with that value somewhere and we don't want to reuse it for something else.